content/providers/05-observability/langfuse.mdx

```diff
@@ -152,6 +152,48 @@ on the [next-openai](https://github.com/vercel/ai/tree/main/examples/next-openai
 
 ## Configuration
 
+### Group multiple executions in one trace
+
+You can open a Langfuse trace and pass the trace ID to AI SDK calls to group multiple execution spans under one trace. The passed name in functionId will be the root span name of the respective execution.
+
+```ts
```

Comment:
do we want to put this in a `main` function for top-level await issues?

Reply:
it's community provided content